terraform/k8s/gitops-build/terragrunt.hcl

```diff
@@ -17,23 +17,23 @@ dependency "k8s_deploy" {
     nat_public_ips                   = [""]
     internal_load_balancer_dns       = ""
     external_load_balancer_dns       = ""
```

Comment:
@kalinkrustev do these specific values need to be added?

Reply:
I am trying to test it and can answer after the tests

Reply:
I have found a way to avoid the use of specific values and after testing it looks to work just fine.
Please recheck the approach in `terraform/k8s/gitops-build/terragrunt.hcl`, where I modified the mocks to take the values from the config file.